HISTORY.md

```diff
@@ -4,6 +4,7 @@ unreleased
   * Set constructor name when possible
   * deps: setprototypeof@1.1.0
   * deps: statuses@'>= 1.3.1 < 2'
+  * deps: depd@1.1.2
```

Comment:
Sorry, I guess I forgot the full rules here, my bad 🤣The list of changes are alphabetical, in the format of "{optional section name}: {message}". All changes without a section name are first, then given the names of the sections, those are sorted and then within each section they are sorted by message.

Reply:
The other addition is that `deps:` will then list in a sub list all the changes that users would notice through the use of this module. There is one change in the `depd` package that they would in this case, the eval -> Function change. So add that as a sub bullet under the depd change 👍